app/views/NewMessageView/ButtonCreate.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { themes } from '../../lib/constants';
+import { CustomIcon, TIconsName } from '../../containers/CustomIcon';
+import { useTheme } from '../../theme';
+import Touch from '../../containers/Touch';
+import sharedStyles from '../Styles';
+import { PADDING_HORIZONTAL } from '../../containers/List/constants';
+
+const styles = StyleSheet.create({
+	button: {
+		height: 46,
```

Comment:
Can you use paddingVertical instead so it's accessible? (won't cut on larger font sizes)